src/commands/common/send-telemetry-command.js

```diff
@@ -43,11 +44,13 @@ class SendTelemetryCommand extends Command {
                 };
                 blockchainsNodeInfo.push(blockchainInfo);
             }
+            const tripleStoreImplementations =
+                this.tripleStoreModuleManager.getImplementationNames();
             const nodeData = {
                 version: pjson.version,
                 identity: this.networkModuleManager.getPeerId().toB58String(),
                 hostname: this.config.hostname,
-                triple_store: this.config.modules.tripleStore.defaultImplementation,
+                triple_stores: tripleStoreImplementations,
```

Comment:
Let's send triple store info the same way we are sending blockchain, as object and field name: ' '